core/includes/bootstrap.inc

```diff
@@ -1011,6 +1016,7 @@ function backdrop_get_filename($type, $name, $filename = NULL) {
  * @deprecated since 1.0
  */
 function variable_initialize($conf = array()) {
+  watchdog_deprecated_function('bootstrap', __FUNCTION__);
```

Comment:
This function is currently *always* called. It's part of the bootstrap process in _backdrop_bootstrap_variables(). We probably shouldn't log an error when there is no way to prevent it.